authorized_keys_command.sh

```diff
@@ -10,6 +10,26 @@ SaveUserName=${SaveUserName//"="/".equal."}
 SaveUserName=${SaveUserName//","/".comma."}
 SaveUserName=${SaveUserName//"@"/".at."}
 
-aws iam list-ssh-public-keys --user-name "$SaveUserName" --query "SSHPublicKeys[?Status == 'Active'].[SSHPublicKeyId]" --output text | while read KeyId; do
-  aws iam get-ssh-public-key --user-name "$SaveUserName" --ssh-public-key-id "$KeyId" --encoding SSH --query "SSHPublicKey.SSHPublicKeyBody" --output text
+# Specify IAM group(s) separated by spaces to import users.
+# Specify "##ALL##" (including the double quotes) to import all users
+IAMGroup=("##ALL##")
```

Comment:
I agree with you. I'm of the same opinion too. The ultimate call is with owner of the repository. I just implemented their suggestion. :wink: